<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomBusinessEventWidget</class>
 <widget class="QWidget" name="CustomBusinessEventWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>131</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
     <item row="0" column="0">
      <widget class="nx::vms::client::desktop::LabelWithHint" name="sourceLabel">
       <property name="text">
        <string>Source contains:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="deviceNameEdit">
       <property name="placeholderText">
        <string>Keywords separated by space</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="nx::vms::client::desktop::LabelWithHint" name="captionLabel">
       <property name="text">
        <string>Caption contains:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="captionEdit">
       <property name="placeholderText">
        <string>Keywords separated by space</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="nx::vms::client::desktop::LabelWithHint" name="descriptionLabel">
       <property name="text">
        <string>Description contains:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="descriptionEdit">
       <property name="placeholderText">
        <string>Keywords separated by space</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="omitLogging">
     <property name="text">
      <string>Omit Logging</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QnWordWrappedLabel" name="hintLabel" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QnWordWrappedLabel</class>
   <extends>QWidget</extends>
   <header location="global">ui/widgets/word_wrapped_label.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>nx::vms::client::desktop::LabelWithHint</class>
   <extends>QLabel</extends>
   <header location="global">nx/vms/client/desktop/common/widgets/widget_with_hint.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
